.github/workflows/publish.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - 'DOI-759-ev-dashboard-client-npm-publish'
+  pull_request:
+    branches:
+      - '*'
```

Comment:
@royki  Does this mean that the publish will run on any pull_request on any branch?

Reply:
We can remove this. This doesn't need actually.